docs/sp-add-ins/create-a-developer-site-on-an-existing-office-365-subscription.md

```diff
@@ -40,24 +40,28 @@ For more information about the Office 365 Enterprise offerings, see [Plans &amp;
 
 4. In the navigation tree on the left, expand **Admin**, and then select **SharePoint**.
 
-5. In the **SharePoint admin center**, on the **Site Collections** tab, select **New** > **Private Site Collection**.
+5. In the **SharePoint admin center**, on the **Sites** menu, select **Active Sites** .
 
-   ![SharePoint Admin Center new site collection option](../images/SPAdminCenter_newSiteCollection.png)
```

Comment:
Please verify if this image is used anywhere else. If not, please remove it as part of this PR